File: server/PredlaganjeSaradnjeIRCDemo/PredlaganjeSaradnjeIRC.Data/Model/RequestForCooperation.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace PredlaganjeSaradnjeIRC.Data.Model
{
    public class RequestForCooperation
    {
        [Required]
        public int Id { get; set; }
        [Required]
        public string Title { get; set; }
        [Required]
        public string DescriptionOfProposal { get; set; }
        [Required]
        public DateTime Date { get; set; }
        public Company Company { get; set; }
        public Employee Employee { get; set; }
    }
}

File: client/fpis-app/src/model/RequestForCooperation.ts
import { Employee } from "./Employee";
import {Company} from './Company';

export class RequestForCooperation{
    id:number;

    title:string;

    descriptionOfProposal:string;

    date:Date;

    company:Company|null;

    employee:Employee|null;

    constructor(id:number,title:string,descriptionOfProposal:string,date:Date,company:Company|null,employee:Employee|null){
        this.id=id;
        this.title=title;
        this.descriptionOfProposal=descriptionOfProposal;
        this.date=date;
        this.company=company;
        this.employee=employee;
    }
}